### PR TITLE
The users who have created the tickets should have access to those ti…

### PR DESCRIPTION
**Pull request by shravan683**
…ckets.

### CODE REVIEW

File: usertickets/UserTicketSystem/UserTicketSystem/settings.py [hunk 1]
```diff
@@ -1,62 +1,3 @@
-"""
```
Comment (ankush-sunedison) [inline usertickets/UserTicketSystem/UserTicketSystem/settings.py:1]: Why are all these lines removed from settings?

File: usertickets/UserTicketSystem/users/models.py [hunk 2]
```diff
@@ -1,12 +1,15 @@
...
-
+from django.contrib.auth import get_user_model
 class CustomUser(AbstractUser):
     pass
 
-class Ticket(models.Model):  
+class Ticket(models.Model):
     title = models.CharField(max_length=100)
     description = models.TextField()
+    user = models.ForeignKey(get_user_model(), on_delete=models.CASCADE)
```
Comment (ankush-sunedison) [inline usertickets/UserTicketSystem/users/models.py:10]: what does get_user_model function do?

File: usertickets/UserTicketSystem/users/views.py [hunk 3]
```diff
@@ -1,43 +1,45 @@
...
-        if serializer.is_valid():
-            serializer.save()
-            return Response(serializer.data)
-        return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+    def get_queryset(self):
+        return Ticket.objects.filter(user=self.request.user)
 
     @action(detail=False, methods=['POST'])
-    def custom_action(self, request, *args, **kwargs):    
+    def custom_action(self, request, *args, **kwargs):
```
Comment (ankush-sunedison) [inline usertickets/UserTicketSystem/users/views.py:62]: what is this method?

File: usertickets/UserTicketSystem/UserTicketSystem/settings.py (hunk 1 above)
Comment (SHRAWANKUMARM) [reply usertickets/UserTicketSystem/UserTicketSystem/settings.py:1]: mistakenly added two times . i have removed duplicate one

File: usertickets/UserTicketSystem/users/views.py [hunk 4]
```diff
@@ -1,43 +1,45 @@
...
         return Response({'message': 'Custom action successful'}, status=status.HTTP_200_OK)
```
Comment (SHRAWANKUMARM) [inline usertickets/UserTicketSystem/users/views.py:43]: Django REST framework viewset custom action

File: usertickets/UserTicketSystem/users/models.py (hunk 2 above)
Comment (SHRAWANKUMARM) [reply usertickets/UserTicketSystem/users/models.py:10]: foregnkey establish relation bwtween user and ticket. user model shows which user is active

File: usertickets/UserTicketSystem/users/views.py (hunk 3 above)
Comment (SHRAWANKUMARM) [reply usertickets/UserTicketSystem/users/views.py:62]: Django REST framework viewset custom action

**Comment by shravan683** [comment]
created the tickets

**Pull request merged**